Added principal styles for buttons (commit test)

BregornOriginal opened the pull request:
This commit was created to show a problem whit buttons.

BregornOriginal committed:
Added principal styles for buttons (commit test)

This commit was created to show a problem whit buttons.

daniNKV commented on simon-says.html:
```diff
@@ -0,0 +1,34 @@
+    <link rel="stylesheet"type="text/css"media="screen"href="simon-says.css"/>
+  </head>
+  <body id="body">
+    <form>
```
No le veo sentido a tener un form, creo que entorpece más en el box model de css que lo que aporta semánticamente

daniNKV commented on simon-says.css:
```diff
@@ -0,0 +1,114 @@
+  top: 185px;
+}
+#color-blue {
+  width: 200px;
```
Todos estos botones tienen propiedades en común. En lugar de reescribirlas para cada uno podés tener una clase base como por ejemplo ".base-button" con las propiedades comunes como, width, height, border, etc. Luego podés agregarle estilos a cada uno por separado con su clase única. Ponele que quedaría el elemento boton con estas clases separadas por un espacio ".base-button boton-1 ". En la clase ".boton-1" agregarias el color y por lo que veo la rotación

BregornOriginal replied:
Tenés razón, entiendo lo que me decis. También estaba revisando los temas de posición, para darle propiedad relativa al padre y poner a todos los botones dentro del base-game y que quede más ordenado.

daniNKV commented on simon-says.css:
```diff
@@ -0,0 +1,114 @@
+  background-color: rgb(53, 5, 5);
+}
+
+#box-container {
```
Guarda con las medidas fixed para ubicar elementos, te terminás volviendo loco luego con los media querys para hacerlo responsive. Con grid y flexbox solucionás casi todo. En el html te dejo una recomendación

BregornOriginal replied:
Gracias por el consejo, lo voy a tener en cuenta!

daniNKV commented on simon-says.html:
```diff
@@ -0,0 +1,34 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet"type="text/css"media="screen"href="simon-says.css"/>
+  </head>
+  <body id="body">
```
Clase padre del juego seria este body, seguramente quieras que esté centrado y con medidas que se vean siempre bien en cualquier pantalla

Con un [display flex ](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)podés centrarlo fácilmente 
y las medidas mínimas o máximas con max y min height/width

BregornOriginal replied:
Sisi. Este tema justamente no llegué a verlo, gracias por anticiparme! Es lo que tenía que revisar justamente.

daniNKV commented on simon-says.html:
```diff
@@ -0,0 +1,34 @@
+    <form>
+        <button id="button-blue"></button>
+        <div></div>
+      <div id="box-container">
```
Esta sería la clase con los botones y el juego, está dentro del body por lo que estaría centrada y respetando las medidas máximas

Los hijos de este elemento son los 4 botones, el botón para iniciar y un elemento del tablero (capaz podría ser más cómodo ponerlo por fuera de este div ya que solo es estético y está más ligado a la clase padre creo). 

Con [grid ](https://css-tricks.com/snippets/css/complete-guide-grid/)podés crear las grillas que quieras, con una grilla de 2x2 podés ubicar a cada botón en un espacio y al botón comenzar en el centro

BregornOriginal committed:
Orderer elements and eliminate form.

In this commit we can see a better order of elements and the form unnecesary element was eliminated.

BregornOriginal committed:
Better adjust of some styles.

Default styles added to #button-color instead of each button color.
